Scripts/developer_scripts/add_license_in_pkg_header.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+# Insert in all the header file of a package the include directive
+# "#include <CGAL/license/${package_name}.h" if it is not already there
+# There are two arguments to the script as some packages are split into
+# subdirectories (Polyhedron_IO, Algebraic_kernel_for_circles, ...)
+
+from sys import argv
+import os
+from os.path import join
+import codecs
+import re
+
+def add_license_include_in_file(package_name, fname):
+  # first, see if the include directive is already there
+  with codecs.open(fname, encoding='utf-8') as f:
+    if any(re.search("CGAL/license/"+package_name+".h", line) for line in f):
```

Comment:
I would search for `#include\s*<CGAL/license/` instead. If `package_name` is, for example, `Mesh_2`, but a file already includes another license checking header, like:

    #include <CGAL/license/Delaunay_conforming_2.h>

then that would be preferable that the file is skipped.